packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -467,13 +462,14 @@ describe('Collection aggregations tab', function () {
 
     // wait for the modal, fill out the modal, confirm
     await duplicateModal.waitForDisplayed();
-    await browser
-      .$(Selectors.DuplicateViewModalTextInput)
-      .setValue('duplicated-view');
+    await browser.setValueVisible(
+      Selectors.DuplicateViewModalTextInput,
+      'duplicated-view'
+    );
     const confirmDuplicateButton = await browser.$(
       Selectors.DuplicateViewModalConfirmButton
     );
-    confirmDuplicateButton.waitForEnabled();
+    await confirmDuplicateButton.waitForEnabled();
```

Comment:
We forgot to wait for this which probably explains the evidence of a screenshot being taken for debugging below 😆